package.json

```diff
@@ -289,6 +293,11 @@
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i",
                     "group": "8@2"
                 },
+                {
+                    "command": "aks.eraserTool",
+                    "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i",
```

Comment:
This tool is applicable to any K8s cluster, not just AKS ones, isn't it? So could we show it in the Cluster Explorer context menu too? That would also make the implementation slightly simpler as well, I think.